### ISSUE TITLE
Add technical definition of outputs

### ISSUE THREAD

**Issue by julianriise** [linkarkitektur/agiliate-engine#1]
Very interesting project, I'm currently blocked by not understanding the output of the model. 
If I understand correctly, the input variables can be set freely, however the output variables (totals) are non-negotiable?

In that case, would you consider adding context or technical definitions to help developers read the output variables (totals):
- Gross area
- Net area
- Adjusted area including compensation with adjustment and compensation
- Unadjusted area
- Workplace area
- Compensation area
- Employees per workplace type unadjusted
- Net area per employee
- Gross area per employee
- Utility floor place per employee
- Gross area per dimensioned attendance
- Gross net factor
- Dimensioned attendance

The outputs might mean different things depending on jurisdiction. This causes confusion to me as a Norwegian developer, as we might use different definitions. A source link to the definitions you have used, or specific explanation would be very helpful I think.

**Comment by huulbaek** [comment]
Yes, this is something we have discussed and will have to look into soon. Actually, the totals are Norwegian terms translated loosely to English.

@JanButhke We should definitely get this done.
